en/docs/fragments/oauth-playground.md

```diff
@@ -4,11 +4,9 @@
 
 ----
 
-(TODO: dev-portal-fragment)
-
 {!fragments/register-playground-application-portal.md!}
-After all the configurations click **Update**.
 
+1.  Click **Update**.
```

Comment:
is it ok to have the number as `1` here?

Reply:
Yes, it gets amended automatically.